src/LogNormal/LogNormalMath.sol

```diff
@@ -22,7 +22,7 @@ function computeTradingFunction(
     LogNormalParams memory params
 ) pure returns (int256) {
     int256 a = Gaussian.ppf(int256(rX.divWadDown(L)));
-    int256 b = Gaussian.ppf(int256(rY.divWadDown(L.mulWadUp(params.mean))));
+    int256 b = Gaussian.ppf(int256(rY.divWadDown(L.mulWadDown(params.mean))));
```

Comment:
Is this meant to be in scope of this PR?

Reply:
I see it in all of the other ones as well

Reply:
(didn't mean to approve, misclick)

Reply:
I think it's a "Git issue", I merged into `main` by mistake and reverted that commit but I think some branches are picking up the non-change as an actual change?